test/data/gplv2/airquality.csv

```diff
@@ -0,0 +1,154 @@
+"","Ozone","Solar_R","Wind","Temp","Month","Day"
+"1",41,190,7.4,67,5,1
+"2",36,118,8,72,5,2
+"3",12,149,12.6,74,5,3
+"4",18,313,11.5,62,5,4
+"5",NA,NA,14.3,56,5,5
+"6",28,NA,14.9,66,5,6
+"7",23,299,8.6,65,5,7
+"8",19,99,13.8,59,5,8
+"9",8,19,20.1,61,5,9
+"10",NA,194,8.6,69,5,10
+"11",7,NA,6.9,74,5,11
+"12",16,256,9.7,69,5,12
```

Comment:
I don't see any usage of this file in PR, not even mentions of air quality.
Is it necessary?
And is it regression dataset?  because we have [https://github.com/dotnet/machinelearning/issues/938](https://github.com/dotnet/machinelearning/issues/938)

Reply:
planning to keep the samples consistent with our other repo. They use this dataset, so it will get used in the next set of samples. Just added the two we need in one shot .

---
In reply to: [228309756](https://github.com/dotnet/machinelearning/pull/1321#discussion_r228309756) [](ancestors = 228309756)